test/functionality/project/package.test.ts

```diff
@@ -6,10 +6,10 @@ describe('DESCRIPTION-file', function() {
 	describe.sequential('Parsing', function() {
 		test('Library-Versions-Plugin', () => {
 			const p1 = new Package('Test Package');
-			p1.addInfo('package', undefined, new Range('>= 1.3'));
-			p1.addInfo(undefined, undefined, new Range('<= 2.3'));
-			p1.addInfo(undefined, undefined, new Range('>= 1.5'));
-			p1.addInfo(undefined, undefined, new Range('<= 2.2.5'));
+			p1.addInfo('package', undefined, undefined, new Range('>= 1.3'));
```

Comment:
again this should be an object with named properties, otherwise this will kill us in the future